packages/expect/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "build"
+    "outDir": "build",
+
+    "stripInternal": true
```

Comment:
To strip `INTERNAL_MATCHER_FLAG`. If it fine here or should this be moved to the root `tsconfig.json`?

Reply:
we can do it in root I think?